app/routers/event.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime
+from typing import Dict, Optional, Set
+
+from app.database.models import Event
+from app.dependencies import templates
+from fastapi import APIRouter, Request
+from sqlalchemy import exc
+from sqlalchemy.orm import Session
+
+router = APIRouter(
+    prefix="/event",
+    tags=["event"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get("/edit")
+async def eventedit(request: Request):
+    return templates.TemplateResponse("eventedit.html", {"request": request})
+
+
+def get_event_by_id(db: Session, event_id: int) -> Event:
+    """Select event by id"""
+    return db.query(Event).filter(Event.id == event_id).first()
+
+
+def check_validation(start_time: datetime, end_time: datetime) -> bool:
+    """Check if the start date is smaller then the end time"""
```

Comment:
You should use start time and not start date and also should be stated as earlier and not smaller.

Reply:
Thanks :)